theano/sandbox/cuda/opt.py

```diff
@@ -1896,7 +1891,7 @@ def local_gpu_downsample_factor_max(node):
             return
         x, = node.inputs
         if (x.owner and isinstance(x.owner.op, HostFromGpu)):
-            gpu_ds = GpuDownsampleFactorMax(**node.op._props_dict())
+            gpu_ds = GpuDownsampleFactorMax(node.op.ds, node.op.ignore_border)
```

Comment:
No. GpuDownsampleFactorMax don't implement all the option/props of Pool. This is why we can't use props here. But we check in line 1888 that we didn't included new props in Pool that this opt don't know about.